src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -84,26 +83,12 @@ public void robotInit() {
 
 		CommandScheduler.getInstance()
 				.onCommandInitialize(
-						command -> {
-							System.out.println("Command initialized: " + command.getName());
-						});
+						command -> System.out.println("Command initialized: " + command.getName()));
 		CommandScheduler.getInstance()
 				.onCommandInterrupt(
-						command -> {
-							System.out.println("Command interrupted: " + command.getName());
-						});
+						command -> System.out.println("Command interrupted: " + command.getName()));
 		CommandScheduler.getInstance()
-				.onCommandFinish(
-						command -> {
-							System.out.println("Command finished: " + command.getName());
-						});
-
-		SmartDashboard.putData(CommandScheduler.getInstance());
-		SmartDashboard.putData(subsystems.drivebaseSubsystem);
-		SmartDashboard.putData(subsystems.armSubsystem);
-		SmartDashboard.putData(subsystems.ledSubsystem);
-		SmartDashboard.putData(subsystems.intakeSubsystem);
-		SmartDashboard.putData(subsystems.visionSubsystem);
```

Comment:
Probably good to keep these in so we have logs of what commands are actually running.

Reply:
I don't think we need to do it twice, though. (See [the lines underneath](https://github.com/robototes/ChargedUp2023/pull/38/files#diff-1ec871e2490d0fa3c6a69284b9b231c43cef77a9aac07f402e4aa729f8fb1018L101-L113))